src/Utilities/useFeatureFlag.js

```diff
@@ -0,0 +1,7 @@
+import { useFlag, useFlagsStatus } from '@unleash/proxy-client-react';
+
+export default (flag) => {
```

Comment:
There is a [hook](https://github.com/RedHatInsights/patchman-ui/blob/6c16d6d18b1c5fb8738b86b3958a8a18008a86da/src/Utilities/Hooks.js#L281C14-L281C28) for the same purpose, but using isBeta flag for some obsolete reason. Let's make a real use of it.

Reply:
Yep I'm removing it and moving the hook to the separate file because it's easier to find and import.